src/config/section/iam.js

```diff
@@ -176,6 +176,17 @@ export default {
         title: 'Accounts',
         param: 'domainid'
       }],
+      tabs: [
```

Comment:
@utchoang @svenvogel: instead of defining a new treeview, you can actually define the component that should render the resource/list view by saying something like:
```
  ...
  component: () => import('@src/components/view/TreeView.vue'),
  ...
```
I'm also fine with current implementation if doing the above change requires a lot of effort.

Reply:
Yes, it will take lots of effort. I think we can go that way later, when we have time and have already completed other tasks with higher priority